task-definition.plugins.json

```diff
@@ -135,7 +135,7 @@
                     "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:STATSD_HOST::"
                 }
             ],
-            "entryPoint": ["./bin/plugin-server", "--no-restart-loop"],
+            "entryPoint": ["./bin/plugin-server-prod"],
```

Comment:
important change

Reply:
Why change this? We'd end up needing to change _all_ of our deployments as well.

Reply:
Check how I made the original file backwards compatible too. So no need to change all deployments.

However, I'd like to start introducing this separation. Handling dev and prod on the same script is a bit iffy and I thought this was more clear

Reply:
Maybe let's introduce the new script into our prod + other deploys separately then? There's IMO risk involved with running plugin server differently in cloud and say helm - good way to introduce bugs, etc.

Reply:
sure, will revert this

Reply:
done, see if the new `bin/plugin-server` makes sense then